### ISSUE TITLE
[RyuJIT/ARM32] Assertion failed 'emitTypeSizes[TypeGet(type)] > 0'

### ISSUE THREAD

**Issue by sjsinju** [dotnet/coreclr#12290]
Message :
```
Assert failure(PID 21787 [0x0000551b], Thread: 21787 [0x551b]): Assertion failed 'emitTypeSizes[TypeGet(type)] > 0' in 'My:Main():int' (IL size 107)
               
                   File: /opt/code/github/hqueue/coreclr/src/jit/emit.h Line: 2377
```
Test
```
Loader/classloader/regressions/Dev12_518401/dev12_518401/dev12_518401.sh
```

**Comment by jyoungyun** [comment]
I'm looking at this issue. :)

**Comment by sjsinju** [comment]
@BruceForstall Please assign this to me.

**Comment by sjsinju** [comment]
Merged #12417
